src/main/java/com/microsoft/jenkins/kubernetes/wrapper/V1ResourceManager.java

```diff
@@ -21,17 +22,35 @@
 import io.kubernetes.client.models.V1Secret;
 import io.kubernetes.client.models.V1Service;
 import io.kubernetes.client.models.V1ServicePort;
+import io.kubernetes.client.util.ClientBuilder;
 
+import java.io.IOException;
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
 
 import static com.google.common.base.Preconditions.checkNotNull;
 
 public class V1ResourceManager extends ResourceManager {
+
+    static {
+        try {
+           STRATEGIC_MERGE_PATCH_CLIENT = ClientBuilder.standard().setOverridePatchFormat(
+                   ApiClient.PATCH_FORMAT_STRATEGIC_MERGE_PATCH).build();
+        } catch (IOException e) {
+            throw new RuntimeException(e.getMessage());
+        }
+
+    }
+
+    private static final ApiClient STRATEGIC_MERGE_PATCH_CLIENT;
     private static final CoreV1Api CORE_V1_API_INSTANCE = new CoreV1Api();
```

Comment:
I found that the client here is a global client. When several tasks are running at the same time, the client may be reset by other tasks. Should I add a client field to the ResourceUpdater?
@gavinfish